pymilvus/orm/collection.py

```diff
@@ -272,7 +272,7 @@ def aliases(self) -> list:
             Aliases of the collection.
         """
         conn = self._get_connection()
-        resp = conn.describe_collection(self._name)
+        resp = conn.describe_collection(self._name, **self._kwargs)
```

Comment:
Do not use `self._kwargs` here. We'd better to customize parameters for every dependent APIs.